pom.xml

```diff
@@ -95,6 +95,9 @@
             <artifactId>a-vaadin-helper</artifactId>
             <version>1.6.0</version>
         </dependency>
+        <!--
```

Comment:
Now when I think about this again, the comment would be even more valuable next to the place where we define which Vaadin version is used, since we should remember to remove this override the next time we touch that part of the file.

Reply:
Changed